Sources/MapboxNavigation/CarPlayManager.swift

```diff
@@ -1144,6 +1144,17 @@ extension CarPlayManager: CPMapTemplateDelegate {
 // MARK: CarPlayNavigationViewControllerDelegate Methods
 
 extension CarPlayManager: CarPlayNavigationViewControllerDelegate {
+    public func carPlayNavigationViewController(_ carPlayNavigationViewController: CarPlayNavigationViewController, 
+                                                waypointCircleLayerWithIdentifier identifier: String,
+                                                sourceIdentifier: String) -> MapboxMaps.CircleLayer? {
+        delegate?.carPlayManager(self, waypointCircleLayerWithIdentifier: identifier, sourceIdentifier: sourceIdentifier)
```

Comment:
Please add unit tests for new methods.
You can take a look at [CarPlayManagerDelegateSpy](https://github.com/mapbox/mapbox-navigation-ios/blob/c09a4b83965809331aaab89271b20566890a57d9/Tests/MapboxNavigationTests/CarPlayUtils.swift#L71) as an example.